webapp/components/reftest-analyzer.js

```diff
@@ -157,75 +167,93 @@ class ReftestAnalyzer extends PolymerElement {
 
   async setupZoomSVG() {
     const zoomed = this.shadowRoot.querySelector('#zoomed');
-    const paths = [];
-    for (let x = 0; x < 5; x++) {
-      paths.push([]);
-      for (let y = 0; y < 5; y++) {
-        const path = document.createElementNS(nsSVG, 'path');
-        const offsetX = x * 50 + 1;
-        const offsetY = y * 50 + 1;
-        path.setAttribute('d', `M${offsetX},${offsetY} H${offsetX + 48} V${offsetY + 48} H${offsetX} V${offsetY}`);
-        path.setAttribute('fill', blankFill);
-        paths[x].push(zoomed.appendChild(path));
+    const pathsBefore = [], pathsAfter = [];
+    for (const before of [true, false]) {
+      const paths = before ? pathsBefore : pathsAfter;
+      for (let x = 0; x < 5; x++) {
+        paths.push([]);
+        for (let y = 0; y < 5; y++) {
+          const path = document.createElementNS(nsSVG, 'path');
+          const offsetX = x * 50 + 1;
+          const offsetY = y * 50 + 1;
+          if (before) {
+            path.setAttribute('d', `M${offsetX},${offsetY} H${offsetX + 48} L${offsetX},${offsetY + 48} V${offsetY}`);
```

Comment:
This is an upper-left half of the square. Below is the lower-right half (cut diagonally).